find_zFrame/find_zFrame.py

```diff
@@ -285,39 +278,34 @@ def onAnteriorChanged(self, newState):
             self.genModel.enabled = True
         else:
             self.fiducialsPresent_list = [i for i in self.fiducialsPresent_list if i!='A']
-        #print("fiducials present: %s"%str(self.fiducialsPresent_list))
     
     def onPosteriorChanged(self, newState):
         if newState == 2:
```

Comment:
To make the code readable, use a constant variable (`newState == qt.Qt.Checked`) instead of magic number (`newState == 2`).